README.md

```diff
@@ -67,9 +67,19 @@ To complete the steps in this tutorial, you need to install Node.js and the Loop
 
    If you don't have a development environment, we'd recommend using [Visual Studio Code](https://code.visualstudio.com/). The LoopBack application generator CLI allows you to generate the VSCode setting files.
 
+## How to use this repo and its branches
+
+This repository contains a master branch with instructions(README.md) and branches for every step with finished code from last step.
+In the workshop, you will git clone this repository and start adding code with the master branch.
+Every time you finish a section, you can run `git checkout <the_next_step>` to begin the next section.
+For example, when following the first section, you will be on branch `master`, after the app is created, you can commit your change(your own implementation), then run `git checkout workshop-part1-complete` which contains our default implementation for section 1 to begin the second section.
```

Comment:
> commit your change 

We don't want them pushing to our repo. Why do we ask them to commit?

Reply:
> This repository contains a master branch with instructions(README.md) and branches for every step with >finished code from last step.
>In the workshop, you will git clone this repository and start adding code with the master branch.>
>Every time you finish a section, you can run `git checkout <the_next_step>` to begin the next section.
>For example, when following the first section, you will be on branch `master`, after the app is created, you can >commit your change(your own implementation), then run `git checkout workshop-part1-complete` which >contains our default implementation for section 1 to begin the second section. 

This repository contains different branches for various stages of this workshop.

The `master` branch only contains the instructions (README.md) for this workshop.

The `workshop-part1-completed` branch contains the instructions (README.md) for this workshop as well as all the artifacts completed in `Part 1` on the workshop. The same is true for the remaining `workshop-part{2,3,4}-completed` branches.


In the workshop, you will `git clone` this repository, and start adding code with the `master` branch.

Every time you finish a `partN` of the workshop, you can run `git checkout <workshop-partN-completed>` before commencing `partN+1`.

Reply:
Does master branch have the instruction?

Reply:
And I thought those branches were for those who haven't finished the previous part

Reply:
@emonddr 
> We don't want them pushing to our repo. Why do we ask them to commit?

**Commit** means save changes on local. **Push** is how the local changes are copied to remote.

@agnes512 

> Does master branch have the instruction?

Yeah master contains all the instructions.

> And I thought those branches were for those who haven't finished the previous part

Hmm, yes or no :-p people finish the section or not, they can always commit the changes and begin the next section with our default code. 
My concern is, user's implementation are usually different than ours, the difference could be minor but sometimes they are critical, e.g. people missed some code in section 1, which doesn't fail the first section but causes the last section fail, then it will be very hard for them to go through the entire tutorial to find the missing part. And that's why I'd like them to start each section with working code from us.

Reply:
I think we can omit the "pushing to local" part in the instruction to avoid confusion. We just need to be clear how to make use of the various branches we've created.

Reply:
@dhmlau removed